analyzers/src/SonarAnalyzer.Common/Helpers/CompilationExtensions.cs

```diff
@@ -31,5 +31,8 @@ public static class CompilationExtensions
             // There's no direct way of checking compilation target framework yet (09/2020).
             // See https://github.com/dotnet/roslyn/issues/3798
             compilation.ObjectType.ContainingAssembly.Name == "mscorlib";
+
```

Comment:
While touching this file, you can move it to "Extensions" directory and change the namespace